packages/gatsby-remark-custom-blocks/README.md

```diff
@@ -2,14 +2,27 @@
 
 Adds custom blocks to `MarkdownRemark` using [remark-custom-blocks](https://github.com/zestedesavoir/zmarkdown/tree/master/packages/remark-custom-blocks).
 
+Unlike
```

Comment:
delete?

Reply:
Whoops, had a train of thought there that went astray. I added a commit to fix this. Let me know if there should be any changes.